CMakeLists.txt

```diff
@@ -51,12 +53,11 @@ target_include_directories(roboteam_utils
 
 target_link_libraries(roboteam_utils
         PUBLIC roboteam_proto
-        #gtest
         )
 target_compile_options(roboteam_utils PRIVATE
         --std=c++17
         -Wall
-        -Werror
+#        -Werror
```

Comment:
The point is we don't want ugly code in this repo. If you need to comment this to make sure it compiles ....

Reply:
So I just compiled locally with the flag and it did not give an error (which is good). I don't see why we should remove it here.

Reply:
So, just change this line back and you should be fine.

Reply:
haha, I still have to look into this. Had warnings locally.

Reply:
will fix

Reply:
already found the issue. The protobuf library does not link for me without warnings. It seems a cannot do something about it. https://github.com/protocolbuffers/protobuf/issues/3128

Reply:
With all due respect I now don't want to change it; with this flag I can't run anything anymore

Reply:
Ah. It's ok for now I guess but let's track that issue and enable it again once it's fixed. That's indeed very silly.